### ISSUE TITLE
Identify disk by ID, not system name

### ISSUE THREAD

**Issue by cluelessperson** [v-zhuravlev/zbx-smartctl#73]
Example 

`/dev/disk/by-id/ata-HGST_<MODEL>_<SERIAL>`

instead of

`/dev/sda`

which changes on reboots and such.

**Comment by rkarlsba** [comment]
I would like this change very much. I've seen spurious false positives after device names change. This is possibly a linux issue, so it should be checked if the machine is running linux first. I'll try to fix it myself, perhaps…

**Comment by rkarlsba** [comment]
I just posted a pull request for this

**Comment by bbrendon** [comment]
Great idea.

**Comment by rkarlsba** [comment]
See pull request #106

**Comment by GabrieleV** [comment]
I suggest by-path, not by-id, because one can change sata hot swap disk daily to take backups.
Same disk path, but different disk id.
Path never changes.

**Comment by cluelessperson** [comment]
Agreed

**Comment by rkarlsba** [comment]
I still like by-id better, since if you were to accidentially reorder the disks, they stay the same. Also, I would never use hotplugging disks for backup. Backups go over the net.

**Comment by GabrieleV** [comment]
I would be nice to have a config entry to choose from by-id, by-path.
We use hotplugging to take backup media offsite manually.

**Comment by Phlogi** [comment]
I fully agree that this pull request is important, as the disk enumartion changes too often between reboots. Also if there is historic data, e.g. temperature, it's difficult to track a specific device when the identifier changes.

**Comment by cluelessperson** [comment]
I noticed this is marked as an "enhancement"  I suggest it's more of a bug.

**Comment by rkarlsba** [comment]
I don't know why this hasn't been pulled. The code is rather straight-forward.
